lib/remote/jsonrpcconnection.cpp

```diff
@@ -76,20 +78,55 @@ void JsonRpcConnection::HandleIncomingMessages(boost::asio::yield_context yc)
 		}
 
 		m_Seen = Utility::GetTime();
+		if (m_Endpoint) {
+			m_Endpoint->AddMessageReceived(jsonString.GetLength());
+		}
+
+		auto start (ch::steady_clock::now());
```

Comment:
... shouldn't this variable be the last of all four?

Reply:
Seriously? How many seconds does the compiler need to declare the variables? :-)

Reply:
You also malloc(3) "UNKNOWN". But I think I've got a nice Idea:

* #10167